google-cloud-spanner/src/test/java/com/google/cloud/spanner/AsyncTransactionManagerImplTest.java

```diff
@@ -60,67 +60,4 @@ public void testCommitReturnsCommitStats() {
       verify(transaction).commitAsync();
     }
   }
-
-  @Test
-  public void testRetryUsesPreviousTransactionIdOnMultiplexedSession() {
```

Comment:
Removing this unit test since it is not possible to change the txnstate of the TransactionManager using unit tests.
Will look at this later if there is a way to fix this test
We have same tests covered in mock spanner https://github.com/googleapis/java-spanner/blob/c8d1e63cb79ee18221dcdad91518842593cc8ded/google-cloud-spanner/src/test/java/com/google/cloud/spanner/MultiplexedSessionDatabaseClientMockServerTest.java#L75